atd-vze/src/routes.js

```diff
@@ -147,7 +148,12 @@ const routes = roles => [
   { path: "/dev/widgets", name: "Widgets", component: Widgets },
   { path: "/dev/charts", name: "Charts", component: Charts },
   { path: "/profile", name: "Profile", component: Profile },
-  { path: "/dashboard", name: "Dashboard", component: VZDashboard },
+  {
+    path: "/dashboard",
+    exact: true,
```

Comment:
This made it so typing in something like /#/dashboard/hello would reroute you back to the dashboard.

Reply:
This works great! I wanted to offer up the idea of also adding a wildcard route to the very end of this array for any other routes not defined in this file that would redirect to the 404 page. Not sure if we would want that behavior, though. It would use a `*` path kind of like [in this article](https://ui.dev/react-router-v5-handling-404-pages).

Reply:
@roseeichelmann this works great as-is! For some reason, I thought there were routes that weren't captured by the redirect behavior so I was thinking that both the wildcard and the redirect behavior would work together to catch all routes. It doesn't seem to be needed though! 🚢🚢🚢